pkg/cmd/config.go

```diff
@@ -82,6 +82,12 @@ func (c *Config) ApplyMachineImageOwnerSecretRef(secretRef **corev1.SecretRefere
 	}
 }
 
+func (c *Config) ApplyWhitelistedImageIDs(whitelistedImageIDs **[]string) {
```

Comment:
why here is using one pointer of pointer of one slice?

Reply:
`options.WhitelistedImageIDs` is `*[]string`, to assign value to it we need to get the address of it.

Reply:
will remove slice pointer